config/pure-python/tox.ini.j2

```diff
@@ -34,9 +34,8 @@ commands =
 {% if with_sphinx_doctests %}
     coverage run -a -m sphinx -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
 {% endif %}
-    coverage html --ignore-errors
-    coverage report --show-missing --fail-under=%(coverage_fail_under)s
+    coverage html
```

Comment:
Maybe also here keep the `--ignore-errors`.

Reply:
see above. All those switches are substituted correctly from `pyproject.toml`.